fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -221,6 +222,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         // DO NOT MOVE ANYTHING ABOVE THIS getProfilerTime CALL.
         val startTimeProfiler = components.core.engine.profiler?.getProfilerTime()
 
+        // Setup nimbus-cli tooling. This is a NOOP when launching normally.
```

Comment:
From the filed issue:

> ..the earliest time the app has access to an intent.

If this means an external intent, then maybe you want this in the `IntentReceivedActivity`? That is earliest time an intent is received.

Reply:
This is CLI sends a `android.intent.category.LAUNCHER` intent with arguments. This is to simulate as closely as possible launching the app by tapping the app icon from the homescreen/springboard/launcher.

Can I/should I send a `LAUNCHER` intent to `IntentReceiverActivity` instead? WDYT.

Reply:
Gotcha, that sounds fine then!